Magento_Catalog/templates/product/view/form.phtml

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Product view template
+ *
+ * @var $block \Magento\Catalog\Block\Product\View
+ */
+?>
+<?php $_helper = $this->helper('Magento\Catalog\Helper\Output'); ?>
+<?php $_product = $block->getProduct(); ?>
+
+<form
+    data-product-sku="<?= /* @NoEscape */ $_product->getSku() ?>"
+    action="<?= /* @NoEscape */ $block->getSubmitUrl($_product) ?>"
+    method="post"
+    class="product-view__form"
+    id="product_addtocart_form"<?php if ($_product->getOptions()): ?>
+    enctype="multipart/form-data"<?php endif; ?>
+>
+    <input type="hidden" name="product" value="<?= /* @escapeNotVerified */ $_product->getId() ?>" />
+    <input type="hidden" name="selected_configurable_option" value="" />
+    <input type="hidden" name="related_product" id="related-products-field" value="" />
```

Comment:
above lines are to long

Reply:
oh c'mon it's only `input type="hidden"` fields, we are not even using it 😄 I always left unseparated hidden fields b/c it's misleading when searching something in template